README.md

```diff
@@ -51,6 +51,18 @@ cmake ..
 make install
 ```
 
+Some special considerations for specific hosts:
+
+###### Archlinux
+
+  Currently, the Clang package distributed via official Pacman sources isn't built with shared libraries, therefore, it is not possible to link
```

Comment:
Remove 'currently' this will continue to be the case.

And that the cause is it doesn't ship with *static* libraries; it is built with shared ones.

Note that the zig cmake flag is required because of an upstream LLVM issue. See https://github.com/ziglang/zig/issues/4799#issuecomment-611886454 for details. The arch bug tracker link isn't appropriate.

Reply:
Thanks for pointers! Updated, have a look and lemme know what you reckon!

Reply:
seems reasonable now :)